settings.py

```diff
@@ -521,7 +521,7 @@ def JINJA_CONFIG():
         # Details: http://jinja.pocoo.org/2/documentation/api#bytecode-cache
         # and in the errors you get when you try it the other way.
         bc = jinja2.MemcachedBytecodeCache(cache._cache,
-                                           "%s:j2:" % settings.CACHE_PREFIX)
+                                           "%s:j3:" % settings.CACHE_PREFIX)
```

Comment:
I realize this probably stands for "jinja2" but I'm cache bumpin' it because :tada: :boom: